packages/docusaurus/src/client/clientEntry.tsx

```diff
@@ -37,8 +37,8 @@ if (ExecutionEnvironment.canUseDOM) {
     </HelmetProvider>
   );
 
-  const onRecoverableError = (error: unknown): void => {
-    console.error('Docusaurus React Root onRecoverableError:', error);
+  const onRecoverableError = (error: unknown, errorInfo: { digest?: string | null, componentStack?: string | null }): void => {
```

Comment:
Thanks, missed that we could get some additional info in the callback

Why not using the official type ReactDOM.ErrorInfo?

Reply:
> the official type ReactDOM.ErrorInfo

Oh. where is that type found?  I used the implementation I found in the React codebase - do you have another link?

Reply:
Doesn't show up in the repo? https://github.com/search?q=repo%3Afacebook%2Freact%20ReactDOM.ErrorInfo&type=code

Reply:
hmm, will try to fix it

Reply:
probably only exists on external typedefs, fixed

Reply:
Oh so the type you're using is from DefinitelyTyped: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/29f3f65685aba9e9b8a105a55494923980e2f16e/types/react/index.d.ts#L3240

Ironically, I'm one of the maintainers of that.  It's slightly inaccurate too :-). But it doesn't really matter - we just want the `console.log` to take place

Reply:
I've raised a PR to update the Definitely Typed definition to be more accurate: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66987

Reply:
yes I know that's why I didn't understand 😅 great if you catched a typo